web/html/src/components/hub/AddTokenButton.tsx

```diff
@@ -86,22 +86,20 @@ export class AddTokenButton extends React.Component<Props, State> {
             title={t("Add a new access token")}
             className="btn-primary"
             items={[
-              <LinkButton
+              // eslint-disable-next-line jsx-a11y/anchor-is-valid
+              <a
```

Comment:
Since we're touching this code, this should be a button rathern than a link since it doesn't navigate anywhere. If we want to keep the look, we can use `className="is-plain"` or other similar options.